File: Programmers/Lv.1/가운데 글자 가져오기/12903.js
function solution(s) {
    let answer = '';
    
    const m = s.length / 2 + 1;
    if (s.length % 2 === 0) {
        answer = s.slice(m - 2, m);
    } else {
        answer = s.slice(m - 1, m);
    }
    
    return answer;
}

File: Programmers/Lv.1/가운데 글자 가져오기/12903.java
class Solution {
    public String solution(String s) {
        String answer = "";
        
        int m = s.length() / 2 + 1;
        if (s.length() % 2 == 0) {
            answer = s.substring(m - 2, m);
        } else {
            answer = s.substring(m - 1, m);
        }
        
        return answer;
    }
}
